Adafruit_FONA.cpp

```diff
@@ -204,12 +277,18 @@ uint8_t Adafruit_FONA::unlockSIM(char *pin)
   return sendCheckReply(sendbuff, ok_reply);
 }
 
+/**
+ * @brief Get the SIM CCID
+ *
+ * @param ccid pointer to ccid buffer
```

Comment:
please Go thru the doxygen again to make sure every function has good documentation.